src/lib/components/Swap/Input.tsx

```diff
@@ -81,11 +82,22 @@ export default function Input({ disabled, focused }: InputProps) {
     return insufficientBalance ? 'error' : undefined
   }, [balance, inputCurrencyAmount, swapInputCurrencyAmount])
 
+  const amount = useInputOutputFieldAmount(
```

Comment:
can we call this `useFormattedFieldAmount` or something? I feel like we have so many vars named Amount rn. Could help to clarify this is a formatting thing